src/java.base/share/classes/sun/security/ssl/CertificateMessage.java

```diff
@@ -501,7 +501,7 @@ private static boolean isIdentityEquivalent(X509Certificate thisCert,
             try {
                 thisSubjectAltNames = thisCert.getSubjectAlternativeNames();
             } catch (CertificateParsingException cpe) {
-                if (SSLLogger.isOn && SSLLogger.isOn("handshake")) {
+                if (SSLLogger.isOn && SSLLogger.isOn("ssl,handshake")) {
```

Comment:
It looks like all of these checks for `SSLLogger.isOn` are unnecessary because the `isOn()` method also checks if logging is enabled. Is this to avoid the method call if the log is disabled?